src/vote-vote/vote-vote.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common';
+import {TypeOrmModule} from "@nestjs/typeorm";
+import {VoteVote} from "./entities/vote-vote.entity";
+import {VoteSubject} from "../vote-subjects/entities/vote-subject.entity";
+import {VoteVoteService} from "./vote-vote.service";
+import {VoteVoteController} from "./vote-vote.controller";
+import {VoteSubjectsService} from "../vote-subjects/vote-subjects.service";
```

Comment:
```suggestion
import { VoteVoteController } from "./vote-vote.controller";
import { VoteSubjectsService } from "../vote-subjects/vote-subjects.service";
```
@Jangyusu 여기에 spacing이 들어가야 할 것 같은데, prettier 설정이 저희끼리 안맞는 것 같습니다~! 여기에만 어노테이션했지만, 동일하게 다른 부분들도 포함입니다~!